Mathlib/Tactic/FunProp/MorExt.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2024 Tomas Skrivan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tomas Skrivan
+-/
+import Std.Data.RBMap.Basic
+
+import Mathlib.Tactic.FunProp.ToStd
+
+/-!
+## `fun_prop` Environment extension that stores all registered morphism coercions.
+-/
+
+namespace Mathlib
+open Lean Meta
+
+namespace Meta.FunProp
+
+
+private local instance : Ord Name := ⟨Name.quickCmp⟩
+
+
+/-- Environment extension storing all morphism coercions. -/
+initialize morCoeDeclsExt : SimpleScopedEnvExtension Name (Std.RBSet Name compare) ←
+  registerSimpleScopedEnvExtension {
+    name := by exact decl_name%
+    initial := .ofArray #[`DFunLike.coe] _
```

Comment:
I'm a bit unsure about this. I need to add `@[fun_prop_coe]` attribute to `DFunLike.coe`. The question is do I hardcode it here or should I add early dependency of fun_prop to `Mathlib.Data.FunLike.Basic` and add the attribute there?

Reply:
I think I'd prefer it hardcoded here. You can add a comment.

Reply:
I followed Eric's recommendation and reused coercion extension already existing in std. So this issue disappeared